database/main.py

```diff
@@ -0,0 +1,31 @@
+# main.py

```

Comment:
I guess we don't need this comment here 🤷🏼  (it is not a big deal though if you want to leave it there).